test/unit/fh3/test_preview.js

```diff
@@ -0,0 +1,35 @@
+var assert = require('assert');
+var genericCommand = require('genericCommand');
+var _ = require('underscore');
+var showPreviewCmd = genericCommand(require('cmd/fh3/preview/show'));
+var urlPreviewCmd = genericCommand(require('cmd/fh3/preview/url'));
+
+var nock = require('nock');
+var data = {
+  "url": "http://henr.ie/2s7DFXD",
+  "longUrl": "https://support-cuke6ultphkoxaulyhpsb6ep-dev.mbaas1.us.feedhenry.com"
+};
+
+var data = "https://support.us.feedhenry.com/box/srv/1.1/wid/support/studio/1a/container";
+
+module.exports = nock('https://apps.feedhenry.com')
```

Comment:
does this need to be exported? Also while not a big deal, I would prefer to see it defined in the test directly so it is explicit and avoids tying both tests to this single definition, this could also be done in a function that the test would call. Again not a big deal though